helm-charts/HPA.md

```diff
@@ -100,60 +100,37 @@ used model, data type and image versions, the specified resource values may need
 
 ### Post-install
 
-Above step created custom metrics config for Prometheus-adapter suitable for HPA use.
+Above step created custom metrics config for Prometheus-adapter, suitable for HPA use.
 
-Take backup of existing custom metrics config before replacing it:
+However, Helm does not allow OPEA chart to overwrite Prometheus-adapter configMap, as
+it belongs to another chart. Therefore manual step is needed to overwrite its current
+custom metric rules.
 
-```console
-$ prom_ns=monitoring  # Prometheus/-adapter namespace
-$ name=$(kubectl -n $prom_ns get cm --selector app.kubernetes.io/name=prometheus-adapter -o name | cut -d/ -f2)
-$ kubectl -n $prom_ns get cm/$name -o yaml > adapter-config.yaml.bak
-```
-
-Save generated config with values matching current adapter config:
+Following will overwrite current Prometheus-adapter custom metric rules with OPEA Helm
```

Comment:
The following will overwrite the current Prometheus adapter custom metric rules with the ones generated by OPEA Helm and will restart the Prometheus adapter to apply the new rules.